parser/src/layout.rs

```diff
@@ -173,6 +173,7 @@ where
         loop {
             // Retrieve the current indentation level if one exists
             let offside = match (&token.value, self.indent_levels.last().cloned()) {
+                (&Token::ShebangLine(_), _) => return Ok(token),
```

Comment:
One of the trickiest parts of the the compiler. But this should be good as shebang should not affect the layouting in any way. 👍